src/Zope2/utilities/zconsole.py

```diff
@@ -15,11 +15,12 @@ def runscript(zopeconf, script_name, *extra_args):
     app.REQUEST['PARENTS'] = [app]
     setRequest(app.REQUEST)
     newSecurityManager(None, user)
-    scriptglobals = globals()
+    scriptglobals = {}
```

Comment:
the next 3 lines could be coded in a single statement

Reply:
+1

Reply:
okay, will do that.